pkg/api/v1/workloads.go

```diff
@@ -219,6 +219,12 @@ func (s *WorkloadRoutes) createWorkload(w http.ResponseWriter, r *http.Request)
 		req.OIDC.Audience,
 		req.OIDC.JwksURL,
 		req.OIDC.ClientID,
+		"",    // otelEndpoint - not exposed through API yet
+		"",    // otelServiceName - not exposed through API yet
+		0.1,   // otelSamplingRate - default value
+		nil,   // otelHeaders - not exposed through API yet
+		false, // otelInsecure - not exposed through API yet
+		false, // otelEnablePrometheusMetricsPath - not exposed through API yet
```

Comment:
nit: it might be good to bundle the OTEL-related properties into a single struct

Reply:
Could this be a subsequent refactoring?